tomviz/operators/OperatorResult.h

```diff
@@ -45,6 +45,10 @@ class OperatorResult : public QObject
   void setLabel(const QString& label);
   const QString& label();
 
+  /// Set description of object.
+  void setDescription(const QString& desc);
+  const QString& description();
```

Comment:
It isn't worth returning as const ref, just return as a value. QString is copy-on-write.